Update README.md - Angular Boilerplate (#42)
aziontech/azion-samples: templates/angular-boilerplate/README.md

# Angular Boilerplate

The **Angular Boilerplate** is designed to simplify and enhance the deployment process for Angular.js applications directly on the edge of the network.

This repository is an example of the new GitHub repository created during the deployment. For a more detailed step-by-step to deploy this template via Azion Real-Time Manager (RTM), check the [How to deploy edge applications with the Angular Boilerplate](https://www.azion.com/en/documentation/products/guides/angular-boilerplate/) guide.

This template uses Angular CLI version `16.1.3`.

---

## Usage information

To use this template, you must [sign in on Azion Cosole](https://console.azion.com/login).

> If you haven't an account, go to the [signup page](https://console.azion.com/signup) to complete the registration.

> To successfully deploy this template, you need to provide the information to configure your Azion application, as shown in the **Settings** tab on the template's page.

Author: ```suggestion
To successfully deploy this template, you need to provide the information to configure your Azion application, as requested in the **Settings** tab on the template's page.
```

> For a more detailed step-by-step on using this template and requirements, check the [How to deploy edge applications with the Angular Boilerplate](https://www.azion.com/en/documentation/products/guides/angular-boilerplate/) guide.

Author: ```suggestion
For a more detailed step-by-step on using this template and requirements, check the [How to deploy edge applications with the Angular Boilerplate](https://www.azion.com/en/documentation/products/guides/angular-boilerplate/) guide.

### Requirements

- Generate a [GitHub personal token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic).
- Enable [Edge Functions](https://www.azion.com/en/documentation/products/guides/billing-and-subscriptions/) on your Azion account.
```

> ## Usage Information

Author: ```suggestion
## Usage information
```

> To successfully deploy this template, you will need to provide the information to configure your Azion application, as shown in the **Setting** tab on the template's page.

Author: ```suggestion
To successfully deploy this template, you will need to provide the information to configure your Azion application, as shown in the **Settings** tab on the template's page.
```

> To successfully deploy this template, you will need to provide the information to configure your Azion application, as shown in the **Settings** tab on the template's page.

Author: ```suggestion
To successfully deploy this template, you need to provide the information to configure your Azion application, as shown in the **Settings** tab on the template's page.
```

# Angular Boilerplate

The **Angular Boilerplate** is designed to simplify and enhance the deployment process for Angular.js applications directly on the edge of the network.

This repository is an example of the new GitHub repository created during the deployment. For a more detailed step-by-step to deploy this template via Azion Real-Time Manager (RTM), check the [How to deploy edge applications with the Angular Boilerplate](https://www.azion.com/en/documentation/products/guides/angular-boilerplate/) guide.

This template uses Angular CLI version `16.1.3`.

---

## Usage information

> To use this template, you must [sign in on Azion Cosole](https://console.azion.com/login).

Author: ```suggestion
To use this template, you must [sign in on Azion Console](https://console.azion.com/login).
```

> > If you haven't an account, go to the [signup page](https://console.azion.com/signup) to complete the registration.

Reviewer: ```suggestion
> If you haven't created an account, go to the [signup page](https://console.azion.com/signup) to complete the registration.
```

Now your project is ready to work with a continuous deployment workflow, updating instantly any changes in the application or the repository. 

---

## Management

> Considering that this initial setup may not be optimal for your specific edge application, all settings can be customized any time you need by using [Azion Real-Time Manager (RTM)](https://manager.azion.com/).

Author: ```suggestion
Considering that this initial setup may not be optimal for your specific edge application, all settings can be customized any time you need by using [Azion Console](https://console.azion.com/).
```
